esp/src/src/nls/bs/hpcc.ts

```diff
@@ -350,6 +357,7 @@
     FileUploadStillInProgress: "Učitavanje Datoteka je u Toku",
     Filter: "Filter",
     FilterDetails: "Opis Filtera",
+    FilterMetricsTooltip: "Ovo će se filtrirati na osnovu kolone "Scope" prema zadanim postavkama. Da biste filtrirali bilo koju drugu kolonu, koristite naziv kolone kao prefiks za pretragu, npr.: "Filename:spill"",
```

Comment:
The double quotes within the string are causing the build to fail.

Reply:
Quotes were corrected.